dbms/src/Storages/DeltaMerge/StableValueSpace.h

```diff
@@ -182,7 +191,7 @@ class StableValueSpace : public std::enable_shared_from_this<StableValueSpace>
          * DTFiles are not fully included in the segment range will be also included in the result.
          * Note: Out-of-range DTFiles may be produced by logical split.
          */
-        const DMFiles & getDMFiles() const { return stable->getDMFiles(); }
+        const DMFiles & getDMFiles() const { return dm_files; }
```

Comment:
Why we need an extra vector of dm_files instead of using the `stable->getDMFIles()`.

And the following `getDMFilesPacks` `getDMFilesRows` `getDMFilesBytes` still use the `stable->xxx`?

Reply:
Removed. This is legacy code of previous implementation. When there is no DTFile on local disk, this piece of code is not needed. We can just use previous code path.